libs/atqr/domain/src/lib/domain-spec/stories/showPopularGoals.disl

```diff
@@ -0,0 +1,19 @@
+domain ATQR
+
+context Challenges
+
+Repository
+  showPopularGoals? :qty
+
+    find challenges with the same goal and group them in a arrayNameGoal
+      for each arrayNameGoal count the lenght
+      order the arrayNameGoal by descending lenght of the array
+
+      then return :qty Goals
```

Comment:
This should be inside the context. Also...

- What is the name of the repository?
- Repositories do not handle commands. You need a service for this command since it manipulates many instances
- You'll still need a repository with a method that has a similar name and does the actual query

Reply:
I dont know if a get it...do you want me to write the repository in this file ?